server.js

```diff
@@ -17,11 +20,57 @@ app.use(require('morgan')('dev'));
 
 // Routes
 app.get('/', function(req, res) {
-  res.send('Hello, backend!');
+  res.render('index')
 });
 
+app.get('/results', (req, res) =>{
+  let search = req.query.search
+  let qs = {
+    params: {
+      s: search,
+      apikey: API_KEY
+    }
+  }
+
+  axios.get('http://www.omdbapi.com', qs)
+  .then((response) =>{
+    console.log(response.data)
+    let movie = response.data.Search
+    //setting a variable to data
+    res.render('results', {data: movie})
+  })
+  .catch(err =>{
+    console.log(err)
+  })
+})
+
+app.get('/movie/:id', (req,res) =>{
```

Comment:
Your results.ejs a tag links to '/movies/:id' so either this route should be '/movies/:id' (plural) or the link in your a tag in your results.ejs page needs to be singular "/movie/<%= t.imdbID %>"